runtime/tests/checker/interface_test.go

```diff
@@ -1991,3 +1992,117 @@ func TestCheckInvalidInterfaceUseAsTypeSuggestion(t *testing.T) {
 		errs[0].(*sema.InvalidInterfaceTypeError).ExpectedType,
 	)
 }
+
+//MultipleInterfaceDefaultImplementation
+func TestCheckInvalidMultipleInterfaceDefaultImplementation(t *testing.T) {
```

Comment:
I guess this is a typo?

```suggestion
func TestCheckInvalidMultipleInterfaceDefaultImplementation(t *testing.T) {
```